package.json

```diff
@@ -19,7 +19,9 @@
     "lint": "eslint .",
     "start": "node src/shared/cli/index.js",
     "qa": "pta ./src/qa/*.qa.js -r log | node src/qa/utils/qaReporter.js",
-    "test": "FILES_MODIFIED=$(git diff --name-only HEAD) jest",
+    "test": "npm run test:jest && npm run test:tape",
+    "test:tape": "tape tests/**/*-test.js | tap-spec",
```

Comment:
We capture modified files for Jest in order to test only the scrapers that have changed. Is this supported by this new test harness?

Reply:
Please see the individual ported test in question; this is a responsibility of the test. Putting it into an env var is very hacky.

Reply:
See my comment for the test in question, we need to diff differently depending on whether we are in a PR or performing regular testing